### PR TITLE
Change Pascalize to remove spaces fix #774

### LINKED ISSUE #774: Pascalize and Camelize do not join words separated by spaces
My expectation of these methods would be that it would join separate words together.

`"This is a sentence".Pascalize();`

Expected result: `"ThisIsASentence"`
Actual result: `"This is a sentence"`

My workaround for now i to `Underscore` first:
`"This is a sentence".Underscore().Pascalize()`

### PR DESCRIPTION
**Pull request by AKTheKnight**
This is a PR to change to fix #774 that changes Pascalize (and therefore Camelize too) to treat spaces in the same way it treats underscores.

I wasn't sure whether to have this as an optional change, or a change to the method itself, so I've chosen to just change the method and create this PR as a way to generate conversation around the issue.

Let me know if you have any thoughts.

### CODE REVIEW

**Comment by onovotny** [comment]
Overall this change is fine, but something seems to have gotten messed up with the readme diff?

**Comment by AKTheKnight** [comment]
Not sure what happened there, think I've fixed it and updated the branch

File: src/Humanizer/InflectorExtensions.cs [hunk 1]
```diff
@@ -67,20 +67,25 @@ public static string Titleize(this string input)
         /// By default, pascalize converts strings to UpperCamelCase also removing underscores
         /// </summary>
         /// <param name="input"></param>
+        /// <param name="removeWhiteSpace">"This has a space" -> "ThisHasASpace" if true</param>
         /// <returns></returns>
         public static string Pascalize(this string input)
         {
-            return Regex.Replace(input, "(?:^|_)(.)", match => match.Groups[1].Value.ToUpper());
+            
+            input = Regex.Replace(input, "(?:^|_| )(.)", match => match.Groups[1].Value.ToUpper());
```
Comment (onovotny) [inline src/Humanizer/InflectorExtensions.cs]: RegEx isn't my strong suit, but is there a way these two regex statements can be combined into one?

Comment (AKTheKnight) [reply src/Humanizer/InflectorExtensions.cs]: I'll have a look what I can come up with

**Comment by AKTheKnight** [comment]
The latest commits remove the second regex line (it wasn't actually needed), and also add support for multiple spaces in a row.

Not sure what the thoughts are on adding support for multiple underscores in a row?

File: src/Humanizer/InflectorExtensions.cs [hunk 2]
```diff
@@ -67,20 +67,25 @@ public static string Titleize(this string input)
...
         /// <param name="input"></param>
+        /// <param name="removeWhiteSpace">"This has a space" -> "ThisHasASpace" if true</param>
         /// <returns></returns>
         public static string Pascalize(this string input)
         {
-            return Regex.Replace(input, "(?:^|_)(.)", match => match.Groups[1].Value.ToUpper());
+            
+            return Regex.Replace(input, "(?:^|_| +)(.)", match => match.Groups[1].Value.ToUpper());
+            
+            //return Regex.Replace(input, @"\s+", "");
```
Comment (onovotny) [inline src/Humanizer/InflectorExtensions.cs]: Please remove the old code and extra spaces in this method.

**Pull request merged**